orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
Rather than installing Hypothesis, wouldn't it be better just to do `sort orders = range(1, n)`?